modules/integration/tests-integration/tests-patches/src/test/resources/artifacts/ESB/nhttp/transport/ESBJAVA4940/axis2.xml

```diff
@@ -0,0 +1,824 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!--
+  ~  Copyright (c) 2005-2010, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
This is the axis2.xml right? So I thought we don't need to alter the years. Is that the correct way?